auth0_flutter/ios/Classes/WebAuth/WebAuthLoginMethodHandler.swift

```diff
@@ -65,6 +65,11 @@ struct WebAuthLoginMethodHandler: MethodHandler {
         if let maxAge = arguments[Argument.maxAge] as? Int {
             webAuth = webAuth.maxAge(maxAge)
         }
+        
+        if let safariViewControllerDictionary = arguments[SafariViewController.key] as? [String: Any?] {
+            let safariViewController = SafariViewController(from: safariViewControllerDictionary)
+            webAuth = webAuth.provider(WebAuthentication.safariProvider(style: safariViewController.presentationStyle ?? UIModalPresentationStyle.fullScreen))
```

Comment:
This default value should be set inside `SafariViewController`'s initializer. See https://github.com/auth0/auth0-flutter/blob/c541f5d3a4c615bca7a003580f9192b4a78f1bf6/auth0_flutter/ios/Classes/CredentialsManager/CredentialsManagerModels.swift#L15 for an example.

Reply:
Fixed in f88d22d1eb2a0f508bc597423792c4940e760f21, let me know if there's a neater way to do this

Reply:
Fixed in f88d22d1eb2a0f508bc597423792c4940e760f21, let me know if there's a neater way to do this